rotkehlchen/tests/unit/test_bitcoin_transactions.py

```diff
@@ -0,0 +1,144 @@
+from typing import TYPE_CHECKING
+
+import pytest
+
+from rotkehlchen.chain.bitcoin.constants import BTC_EVENT_IDENTIFIER_PREFIX
+from rotkehlchen.chain.bitcoin.types import string_to_btc_address
+from rotkehlchen.constants.assets import A_BTC
+from rotkehlchen.fval import FVal
+from rotkehlchen.history.events.structures.base import HistoryEvent
+from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.tests.utils.bitcoin import get_decoded_events_of_bitcoin_tx
+from rotkehlchen.types import BTCAddress, Location, TimestampMS
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.bitcoin.manager import BitcoinManager
+
+
+@pytest.mark.vcr
+@pytest.mark.parametrize('btc_accounts', [
+    ['bc1qw508d6qejxtdg4y5r3zarvary0c5xw7kv8f3t4'],
+    ['1G3MiaKdccQmiTr4gYSKmrCVDaLQ5nvBRp'],
+    ['bc1qw508d6qejxtdg4y5r3zarvary0c5xw7kv8f3t4', '1G3MiaKdccQmiTr4gYSKmrCVDaLQ5nvBRp'],
+])
+def test_1input_1output(bitcoin_manager: 'BitcoinManager', btc_accounts: list[BTCAddress]) -> None:
+    tx_id, address1, address2 = (
+        'e47f43692083b6b4bb3d4d6150acd3c016b09fb841e4055e1f5bb8ad44858bc6',
+        string_to_btc_address('bc1qw508d6qejxtdg4y5r3zarvary0c5xw7kv8f3t4'),
+        string_to_btc_address('1G3MiaKdccQmiTr4gYSKmrCVDaLQ5nvBRp'),
+    )
+    events = get_decoded_events_of_bitcoin_tx(bitcoin_manager=bitcoin_manager, tx_id=tx_id)
+    gas_event = HistoryEvent(
+        event_identifier=(event_identifier := f'{BTC_EVENT_IDENTIFIER_PREFIX}{tx_id}'),
+        sequence_index=0,
+        timestamp=(timestamp := TimestampMS(1686238076000)),
+        location=Location.BITCOIN,
+        event_type=HistoryEventType.SPEND,
+        event_subtype=HistoryEventSubType.FEE,
+        asset=A_BTC,
+        amount=(fee_amount := FVal('0.00002492')),
+        location_label=address1,
+        notes=f'Burn {fee_amount} BTC for gas',
+    )
+    if btc_accounts == [address1]:  # only input tracked - gas event and spend event
```

Comment:
adjust terminology here and elsewhere

```suggestion
    if btc_accounts == [address1]:  # only input tracked - fees event and spend event
```